recipe/meta.yaml

```diff
@@ -38,11 +38,10 @@ requirements:
     - jedi >=0.16
     - matplotlib-inline
     - pexpect >4.3    # [unix]
-    - pickleshare
     - prompt-toolkit >=3.0.41,<3.1.0
     - pygments >=2.4.0
     - stack_data
-    - traitlets >=5
+    - traitlets >=5.13.0
```

Comment:
This was updated in `ipython` version `8.22.1` with PR: https://github.com/ipython/ipython/pull/14353

Reply:
We'll want to follow up with a `repodata-patches` PR for `.0`, or the solver may random-walk to the older `traitlets`. I haven't done any forensics as to where this creates an issue in downstreams, but seems easy enough in this case.

Reply:
Agreed. @jakirkham, would you be able to help with this?

Reply:
It looks like Nick started PR: https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/671

Would be happy to review 🙂